src/Couchbase.Lite.Shared/Sync/WebSocketWrapper.cs

```diff
@@ -105,7 +105,7 @@ internal sealed class WebSocketWrapper
     {
         #region Constants
 
-        private const uint MaxReceivedBytesPending = 100 * 1024;
+        private const uint MaxReceivedBytesPending = 100 * 850;
```

Comment:
Is this still needed even with the other changes?  I don't really want to reduce this amount since it has an effect on performance of the network connection.

Reply:
I guess it's a trade off between space vs speed.. Let me revert it.